packages/nc-gui/components/dlg/QuickImport.vue

```diff
@@ -270,6 +270,17 @@ function getAdapter(name: string, val: any) {
 defineExpose({
   handleChange,
 })
+
+/** a workaround to override default antd upload api call */
+const customReqCbk = (o) => {
```

Comment:
Isn't eslint complaining that `o` is not typed? It should

Also what is `o`? A more descriptive argument name would help discern what it means